test/benchgc/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+################################################################################
+# Copyright (C) 2024 Intel Corporation
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+if(NOT GC_BENCH_ENABLE)
+    message(STATUS "Benchgc is not enabled")
+    return()
+endif()
+
+configure_file(setup.py ${CMAKE_BINARY_DIR}/test/benchgc/setup.py COPYONLY)
+
+file(GLOB PYTHON_SCRIPTS "*.py")
+foreach(PY_SCRIPT ${PYTHON_SCRIPTS})
+  configure_file(${PY_SCRIPT} ${CMAKE_BINARY_DIR} COPYONLY)
+endforeach()
+
+add_custom_target(benchgc
```

Comment:
Is the installation of benchgc enabled as a default target of cmake? I think it should only be triggered when the user requests to.

Reply:
@Menooker 
I'm not quite sure about this. The configuration `-DGC_BENCH_ENABLE` is default `ON`
It behaves different between `make` and `ninja`. `ninja` will not build `benchgc` as default, but `make` will do.
I'm curious about how to control it in `cmake`

Reply:
If running `ninja` will not build `benchgc` by default, that is what I wanted. We are good here. :)